rvuejs/.gitignore

```diff
@@ -0,0 +1,25 @@
+# Logs
```

Comment:
Could you remove the entire `rvuejs` directory.

Reply:
done